Formula/chunkwm.rb

```diff
@@ -0,0 +1,108 @@
+
+    # install chunkwm
+    system "make", "install"
+    inreplace "#{buildpath}/examples/chunkwmrc", "~/.chunkwm_plugins", "$(brew --prefix chunkwm)/share/plugins"
+    bin.install "#{buildpath}/bin/chunkwm"
+    (share/"examples").install "#{buildpath}/examples/chunkwmrc"
+
+    # install chunkc
+    system "make", "--directory", "src/chunkc"
+    bin.install "#{buildpath}/src/chunkc/bin/chunkc"
+
+    # install tiling plugin
+    if build.with? "tiling"
+      system "make", "install", "--directory", "src/plugins/tiling"
+      (share/"plugins").install "#{buildpath}/plugins/tiling.so"
+      (share/"examples").install "#{buildpath}/src/plugins/tiling/examples/khdrc"
+    end
+
+    # install ffm plugin
+    if build.with? "ffm"
+      system "make", "install", "--directory", "src/plugins/ffm"
+      (share/"plugins").install "#{buildpath}/plugins/ffm.so"
+    end
+
+    # install border plugin
+    if build.with? "border"
+      system "make", "install", "--directory", "src/plugins/border"
+      (share/"plugins").install "#{buildpath}/plugins/border.so"
+    end
+
+    # install transparency plugin
+    if build.with? "transparency"
+      system "make", "install", "--directory", "src/plugins/transparency"
+      (share/"plugins").install "#{buildpath}/plugins/transparency.so"
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    Copy the example configuration into your home directory:
+      cp #{share}/examples/chunkwmrc ~/.chunkwmrc
+
+    Opening chunkwm will prompt for Accessibility API permissions. After access has been granted, the application must be restarted.
+      brew services restart chunkwm
+
+    This has to be done after every update to chunkwm, unless you codesign the binary with self-signed certificate before restarting
+      Create code signing certificate named "chunkwm-cert" using Keychain Access.app
+      codesign -fs "chunkwm-cert" #{bin}/chunkwm
+
+    For keybindings install and configure https://github.com/koekeishiya/khd.
+    chunkwm provides an example khd configuration file for easier setup:
+      cp #{share}/examples/khdrc ~/.khdrc
+    EOS
+  end
+
+  plist_options :startup => false
+
+  def plist; <<-EOS.undent
+        <?xml version="1.0" encoding="UTF-8"?>
+        <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+        <plist version="1.0">
+        <dict>
+          <key>Label</key>
+          <string>com.koekeishiya.chunkwm</string>
+          <key>ProgramArguments</key>
+          <array>
+                <string>#{bin}/chunkwm</string>
+          </array>
+            <key>EnvironmentVariables</key>
+            <dict>
+            <key>PATH</key>
+            <string>/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+            </dict>
+          <key>RunAtLoad</key>
+          <true/>
+          <key>KeepAlive</key>
+          <true/>
+            <key>StandardOutPath</key>
+            <string>/tmp/chunkwm.out</string>
```

Comment:
Had to revert this to `/tmp`, Homebrew can't use the suggested path unless a user already created it.